content/docs/0.8.1/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -112,6 +113,13 @@ We provides two testing purpose backupstore based on NFS server and Minio S3 ser
 
 **Result:** Longhorn can store backups in S3. To create a backup, see [this section.](../create-a-backup)
 
+### Using a self signed SSL certificate for S3 communication
+If you want to use a self signed ssl certificate you can expose an `AWS_CERT` environment variable that contains your public certificate to the longhorn pods.
```

Comment:
Is there another way to use custom CA other than set it in the secret? If not, we don't need to mention how we implemented it.